test/unit/utility/views/CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 add_subdirectories()
 
 seqan3_test (enforce_random_access_test.cpp)
+seqan3_test(interleave_test.cpp)
+seqan3_test(pairwise_combine_test.cpp)
 seqan3_test (repeat_n_test.cpp)
 seqan3_test (repeat_test.cpp)
 seqan3_test (single_pass_input_test.cpp)
```

Comment:
Can you remove all the other unnecessary white spaces?
so it is `seqan3_test(file.cpp)` (I checked there are 240 without whitespace and 60 entries with whitespaces)